test/helpers/test-case.js

```diff
@@ -0,0 +1,27 @@
+const header = 'const test = require(\'ava\');';
+
+function testCaseBuilder(ruleId, options = []) {
+	return function testCase(content, errorMessage = [], withoutHeader = false) {
+		const testFn = `
+			test(t => {
```

Comment:
The test should be named. AVA requires this.
```suggestion
			test('x', t => {
```